test.browser-bootstrap.js

```diff
@@ -0,0 +1,40 @@
+const { spawn } = require('child_process')
+const puppeteer = require('puppeteer')
+
+function isOldNode () {
```

Comment:
Wouldn't it be more elegant to handle this via github workflows, separating airtap tests into separate workflow, only running on supported Node versions?

Reply:
Six of one, half a dozen of the other. Separating it at GitHub workflows is both more elegant, but less convenient when hypothetically an engineer runs the tests locally on an unsupported Node versions or if you change CI providers in the future.

Feel free to use your judgement call.

Reply:
I would argue that running tests on obsolete Node locally is on your own peril, tests shouldn't obscure the fact that they are not working. And if those two are separate npm scripts, there is no inconvenience at all, I would actually prefer to see them failing explicitly should I decide to run them.

Reply:
We use separate workflows pretty intensively on other `fastify` repos, optimizing against CI provider lock-in is not something we are actively pursuing right now.

Reply:
Implemented:

![image](https://user-images.githubusercontent.com/113721/104986304-71839e00-5a1b-11eb-8b8c-e4a51980836a.png)

Continuing conversation for curiosity’s sake, feel free to disengage.

You have mentioned valid judgement calls, but there are also reasons to choose otherwise. For instance, one may consider the “test” command to be the “improve my life by automatically validating as much as you can”. Failing on an old version is not helping, but is being a nuisance.

Note that the tool `standard` that is already a part of this project does choose to skip running on older Nodes: `standard: Node 10.12.0 or greater is required. `standard` did not run.` ([CI run](https://github.com/fastify/secure-json-parse/pull/21/checks?check_run_id=1725375426#step:5:8))

I totally agree with the value of your preference, and my own judgement is often aligned with yours, but ultimately, this is a trade-off to be decided on a case-by-case basis. It is impossible to make these decisions from outside of the project and at some point you may want to consider such decisions to be made in the context of some code already being in a PR and the cost of rework associated with that. I don’t think we’ve reached this point yet, but the more we move into preferences the closer we get.